system/metrics/sinks/elastic/elasticSink.cpp

```diff
@@ -291,6 +298,10 @@ void ElasticMetricSink::intializeElasticClient()
     // Add cert path if needed
     if (!certificateFilePath.isEmpty())
         pClient->set_ca_cert_path(certificateFilePath.str());
+
+    pClient->set_connection_timeout(5);
```

Comment:
default timeout values are always difficult and usually don't cover most situations. A couple of suggestions:
- make these configurable
- connect and read/write timeout values should be independent, 5 sec connection timeout might be reasonable, but successful reads/writes might take longer than 5 secs